src/libraries/System.Private.CoreLib/src/System/Text/StringBuilder.cs

```diff
@@ -1232,6 +1233,28 @@ public StringBuilder Append(ReadOnlySpan<char> value)
 
         public StringBuilder Append(ReadOnlyMemory<char> value) => Append(value.Span);
 
+        /// <summary>Appends the specified interpolated string to this instance.</summary>
+        /// <param name="builder">The interpolated string to append.</param>
+        /// <returns>A reference to this instance after the append operation has completed.</returns>
+        public StringBuilder Append([InterpolatedBuilderArgument((string?)null)] ref InterpolatedAppendBuilder builder) => this;
+
+        /// <summary>Appends the specified interpolated string to this instance.</summary>
+        /// <param name="provider">An object that supplies culture-specific formatting information.</param>
+        /// <param name="builder">The interpolated string to append.</param>
+        /// <returns>A reference to this instance after the append operation has completed.</returns>
+        public StringBuilder Append(IFormatProvider? provider, [InterpolatedBuilderArgument(null, "provider")] ref InterpolatedAppendBuilder builder) => this;
```

Comment:
I get it.  But, wow.  That looks broken :smile:.

Reply:
Yeah.